README.md

```diff
@@ -1,3 +1,3 @@
-# I attempt to input the apostrophe character.
+# I attempt to input the apostrophe character using my keyboard which is plugged into my Personal Computer.
 
-It proceeds to throw an error when I do as such.
+It proceeds to throw a fatel error when I do as such.
```

Comment:
_Fatel_ means the "movement of the planets that decides the fortune of the person." I'm pretty sure you meant fatal.